frontend/dispatcher.lua

```diff
@@ -273,19 +272,15 @@ local settingsList = {
     kopt_doc_language = {category="string", paging=true},
     kopt_forced_ocr = {category="configurable", paging=true},
     kopt_writing_direction = {category="configurable", paging=true},
-    kopt_defect_size = {category="string", paging=true},
-    kopt_detect_indent = {category="configurable", paging=true},
+    kopt_defect_size = {category="string", paging=true}, -- not shown in the bottom menu
     kopt_max_columns = {category="configurable", paging=true},
     kopt_auto_straighten = {category="absolutenumber", paging=true},
-
-    settings = nil, -- reserved for per instance dispatcher settings
```

Comment:
This is there for documentation purposes

Reply:
Restored.
I don't find this line to be helpful.